src/remoteConnector.ts

```diff
@@ -760,12 +345,12 @@ export class RemoteConnector extends Disposable {
 							this.logService.error('No SSH gateway:', e);
 							const ok = 'OK';
 							await this.notificationService.showWarningMessage(`${e.host} does not support [direct SSH access](https://github.com/gitpod-io/gitpod/blob/main/install/installer/docs/workspace-ssh-access.md), connecting via the deprecated SSH tunnel over WebSocket.`, { flow: gatewayFlow, id: 'no_ssh_gateway' }, ok);
```

Comment:
The message is wrong now. If both Local SSH and SSH Gateway we should say something like `failed to connect to the workspace` and point out how troubleshooting that users can report bugs or inspect logs

Reply:
We should not point to https://github.com/gitpod-io/gitpod/blob/main/install/installer/docs/workspace-ssh-access.md anymore since we don't have self hosted.

But rather point to Local SSH since it works over HTTP, we should not use local SSH is in user faced messages or proxy or so